src/BookNavigator/downloads/downloads.js

```diff
@@ -59,12 +69,24 @@ export class IABookDownloads extends LitElement {
     `;
   }
 
+  get installSimplyEAldikoThoriumMsg() {
+    return html`
+    <p>For LCP downloads, make sure you have SimplyE or Aldiko Next installed on mobile or Thorium on desktop.</p>
+    <ul>
+      <li><a href="https://librarysimplified.org/simplye/" rel="noopener noreferrer" target="_blank">Install SimplyE</a></li>
+      <li><a href="https://www.demarque.com/en-aldiko" rel="noopener noreferrer" target="_blank">Install Aldiko</a></li>
+      <li><a href="https://www.edrlab.org/software/thorium-reader/" rel="noopener noreferrer" target="_blank">Install Thorium</a></li>
```

Comment:
Did you thought about `nofollow` as well inside rel attribute?

Reply:
thanks for the catch. updated.